config/openshift/20-monitoring.yaml

```diff
@@ -63,4 +63,26 @@ spec:
     - pipelines-as-code
   selector:
     matchLabels:
-      app: pipelines-as-code-watcher
\ No newline at end of file
+      app: pipelines-as-code-watcher
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: pipelines-as-code-controller-monitor
+  namespace: pipelines-as-code
+  labels:
+    app.kubernetes.io/version: "devel"
+    app.kubernetes.io/part-of: pipelines-as-code
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+spec:
+  endpoints:
+    - interval: 10s
```

Comment:
sounds good to me! but i don't know much about this can you explains what it does ? (in the pr description AND commit)

Reply:
I have commented the requirement [here](https://issues.redhat.com/browse/SRVKP-3244?focusedId=22842496&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22842496)

Reply:
in the pull request description and the commit you have sent over

Reply:
Added in the PR description